docs/partner-integrations/partner-integrations.md

```diff
@@ -0,0 +1,10 @@
+![Optimizely logo](./assets/optimizely.svg) | In this exercise we will define, launch, and evaluate the results of an A/B experiment of a personalized user experience using [Optimizely](https://www.optimizely.com/). | [AB Experiments for Personalization](https://github.com/aws-samples/retail-demo-store/blob/master/workshop/3-Experimentation/3.6-Optimizely-AB-Experiment.ipynb) | 200 | 30 minutes
+![Segment logo](./assets/segment.svg) | [Segment](https://segment.com/) is a real-time events pipeline for customer data, as well as a customer data platform.  In the Retail Demo Store, Segment is used to deliver real-time events from the web user interface to Amazon Personalize.  These real-time events are also used to create customer profile with Amazon Personalize recommendations appended, which can then be used via the CDP to push data to marketing tools. | [Real Time Personalization Events](https://github.com/aws-samples/retail-demo-store/blob/master/workshop/1-Personalization/Lab-5-Real-time-events-Segment.ipynb) <br/><br/> [Customer Data Platforms and Personalize](https://github.com/aws-samples/retail-demo-store/blob/master/workshop/6-CustomerDataPlatforms/6.1-Segment.ipynb) | 300 | 1-1.5 hours
+![Layer0 logo](./assets/Layer0.svg) | [Layer0](https://layer0.co/) extends the capabilities of a traditional CDN by not only hosting your static content, but also providing server-side rendering for progressive web applications. Layer0 allows caching both your APIs and HTML at the network edge to provide your users with the fastest browsing experience. Teams can ship faster leveraging an enhanced developer experience to deploy code faster and with more frequency, view their code quickly in atomically deployed environments, and integrating their CDN configuration to the overall build process. Layer0 provides the tools needed to build the modern apps capable of providing the performance expected by modern consumers.  | [Edge Optimization](https://github.com/aws-samples/retail-demo-store/blob/master/workshop/8-PerformanceOptimization/8.1-Layer0.ipynb) | 200 | 1 hour
```

Comment:
Should add an indicator to highlight that this is currently going through a refresh and unlikely to work